docs/installation/linux/debian.md

```diff
@@ -132,7 +132,7 @@ Before installing Docker, make sure you have set your `APT` repository correctly
 
 5. Start the `docker` daemon.
 
-        $ sudo service docker start
+        $ sudo systemctl start docker
```

Comment:
Debian definitely has a `service` wrapper that automatically chooses between sysvinit and systemd -- source is at https://sources.debian.net/src/sysvinit/2.88dsf-59/debian/service/service/ (packaged in [the `sysvinit-utils` package](https://packages.debian.org/jessie/sysvinit-utils), which is still "essential" and thus part of every Debian install), and since we still support both Wheezy and Jessie (and the Debian project still actively supports sysvinit) I think we should keep using it unless there's a specific reason not to (such as systemd-specific functionality, or `/usr/sbin/service` being removed from the base Debian install).